src/routes/safe/components/Settings/SafeDetails/index.jsx

```diff
@@ -40,11 +45,32 @@ const ChangeSafeName = (props: Props) => {
     showSnackbar(notification.afterExecution.noMoreConfirmationsNeeded, enqueueSnackbar, closeSnackbar)
   }
 
+  React.useEffect(() => {
+    const getVersion = async () => {
+      const safeMaster = await getSafeMasterContract()
+      const current = await safeMaster.VERSION()
+      const latest = process.env.REACT_APP_LATEST_SAFE_VERSION
+      const needUpdate = semverLessThan(current, latest)
+
+      setSafeVersions({ current, latest, needUpdate })
+    }
+    getVersion()
+  }, [])
+
   return (
     <>
       <GnoForm onSubmit={handleSubmit}>
         {() => (
           <>
+            <Block className={classes.formContainer}>
+              <Heading tag="h2">Safe Version</Heading>
+              <Row align="end" grow>
+                <Block className={classes.root}>
+                  {safeVersions.current}
```

Comment:
Here the same, if the data is not defined it looks like this:
![image](https://user-images.githubusercontent.com/21086218/71026349-d289c580-20e7-11ea-9790-f4c1f75de274.png)

Maybe you could replace `0.0.0` with `-` or `Not Found` or something else